Readme.md

```diff
@@ -237,6 +241,14 @@ Set the quickstart to not include any oh-my-zsh plugins from the standard plugin
 
 Sets the quickstart to include the oh-my-zsh plugins from the standard plugin list.
 
+##### zqs enable-ssh-askpass-require
+
+Enable the quickstart to prompt for your ssh passphrase on the command line.
+
+##### zqs diasble-ssh-askpass-require
+
+The quickstart will prompt for your ssh passphrase via a gui program. Default behavior.
```

Comment:
Should there be an option to set a specific gui program to use, or just let `ssh` handle it and configure it there?

Reply:
Well that is a good question. On my Linux install it asks for the default mentioned in the manpage for `ssh-add` in my commit [message above](https://github.com/unixorn/zsh-quickstart-kit/pull/229#issue-1411979529). :point_up: 

When I got the error before it was looking for `/usr/bin/ssh-askpass`. Should the kit prompt them to install that package if they see that message or not? What do you think?

Reply:
Maybe have it check for `ssh-askpass` with `can_haz` when they run `zqs diasble-ssh-askpass-require` and prompt them if it's missing. That way it won't have to check every time they start a new shell.

Reply:
I meant when they run `zqs enable-ssh-askpass-require`. Wrap it all in a function so we can keep the dispatch table in `zqs` nice and short.

Reply:
I have now reworked the feature using functions like you mentioned.  I am not sure about the location of the initial `ssh-askpass` check if it should be inside or outside `load-our-ssh-keys()`.

What do you think?